packages/blocks/src/CookieBanner/CookieBanner.tsx

```diff
@@ -61,14 +61,10 @@ export const CookieBanner: FC<CookieBannerProps> = ({
       {...rest}
     >
       <Alert variant="subtle" pt="0" pr={{ base: '0', lg: '2rem' }} pb={{ base: '2rem', lg: '0' }}>
+        <Alert.Icon />
```

Comment:
I have updated this file to avoid chromatic from complaining, but I am not sure if we need to create a new version first of `components` package and then update `blocks` with this new version and update `CookieBanner`